test/Solver/Z3LargeConstantArray.kquery

```diff
@@ -0,0 +1,11 @@
+# REQUIRES: z3
+# RUN: %kleaver -solver-backend=z3 -debug-z3-dump-queries=%t.smt2  %s &> /dev/null
+# RUN: grep '(assert (= (select const_arr101' %t.smt2 -c | grep 3770
```

Comment:
Please don't use `grep`. Use `FileCheck` instead.

Reply:
Can you do that will `FileCheck`, that is assert there are exactly `3770` matches of the pattern. I don't find the documentation for FileCheck particularly clear, so I don't know how to use it well.

Reply:
Ah I missed that you were counting the number of constant array assertions. FileCheck is not good for something like this. However I think you would be much better off making the query use a much smaller constant array and then using FileCheck to make sure that all the asserts are exactly as they should be. You can use `CHECK-DAG` to ignore the order than asserts are made in.

Reply:
I agree adding a test with small array makes sense, but I think it's good to keep this one, because it illustrates the performance difference between the two approaches in Z3.